app/service/dependency/sqlite3.js

```diff
@@ -11,8 +12,8 @@ async function checkInstall() {
     await FS.access(Path.join(ep, 'node_modules/sqlite3/LICENSE'))
   } catch(err) {
     console.warn('未安装 sqlite3 驱动')
-    const result = await noty.warn('未安装 sqlite3 驱动，你需要先安装，才可以连接 sqlite3 数据库', ['安装指引'])
-    if(result == '安装指引')
+    const result = await noty.warn(vscode.l10n.t('SQLite3 driver is not installed, please install SQLite3 driver'), [vscode.l10n.t('Install guide')])
```

Comment:
The SQLite3 driver is not installed, you will need to install it before you can connect to the SQLite3 database